waitress/app/viewsets.py

```diff
@@ -65,6 +65,19 @@ def trim_users(self, request):
 
         return Response(content, status=status_code.HTTP_200_OK)
 
+    @list_route(methods=['post'], url_path='add-guest')
```

Comment:
@list_route is a `django rest framework` decorator for defining a route that encapsulates a list view.